packages/benchmark.rb

```diff
@@ -52,15 +22,15 @@ def self.build
               -DBENCHMARK_ENABLE_GTEST_TESTS=ON \
               -DBENCHMARK_ENABLE_TESTING=ON \
               -DINSTALL_GTEST=OFF .."
```

Comment:
Why do we need gtest source code if we're turning it off?

Reply:
Just trust me on this one.

Reply:
You definitely lost me.